integration/http/src/test/java/io/seata/integration/http/HttpTest.java

```diff
@@ -0,0 +1,189 @@
+/*
+ *  Copyright 1999-2019 Seata.io Group.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+package io.seata.integration.http;
+
+import com.alibaba.fastjson.JSON;
+import com.alibaba.fastjson.JSONObject;
+import com.google.common.collect.Maps;
+import io.seata.core.context.RootContext;
+import org.apache.http.HttpResponse;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.ByteBuffer;
+import java.nio.channels.Channels;
+import java.nio.channels.ReadableByteChannel;
+import java.nio.channels.WritableByteChannel;
+import java.util.Map;
+
+import static io.seata.integration.http.AbstractHttpExecutor.convertParamOfBean;
+import static io.seata.integration.http.AbstractHttpExecutor.convertParamOfJsonString;
+
+/**
+ * @author wangxb
+ */
+class HttpTest {
+
+    private static final String host = "http://127.0.0.1:8081";
+    private static final String getPath = "/index";
+    private static final String postPath = "/testPost";
+    private static final String XID = "127.0.0.1:8081:87654321";
+    private static final int PARAM_TYPE_MAP = 1;
+    private static final int PARAM_TYPE_BEAN = 2;
+
+    @Test
+    void testGetProviderXID() {
+        RootContext.bind(XID);
+        providerStart();
+        consumerGetStart(PARAM_TYPE_MAP);
```

Comment:
How to confirm that the xid was passed to the provider correctly?

Reply:
Ok，I add print code in provider method